pixel.js

```diff
@@ -126,6 +127,7 @@ function setupCli() {
 		.description( 'Create reference (baseline) screenshots and delete the old reference screenshots.' )
 		.requiredOption( ...branchOpt )
 		.option( ...changeIdOpt )
+		.option( '--mobile' )
```

Comment:
Can a description of the option be added (second param)?

Reply:
Thinking about this a bit more how about we make this more generic e.g. --group mobile.

That would allow us to have specific scenarios for testing specific features.. thoughts?

Reply:
Using `--group` sounds good to me! I like the idea to make it generic

Reply:
Done!